Dockerfile

```diff
@@ -1,93 +1,74 @@
 
 # Requisites for ARM
 # ARM EABI toolchain must be grabbed from an contributor repository, such as:
 # https://copr.fedoraproject.org/coprs/lantw44/arm-linux-gnueabi-toolchain/
-RUN yum install -y yum-utils && yum-config-manager --add-repo=https://copr.fedoraproject.org/coprs/lantw44/arm-linux-gnueabi-toolchain/repo/fedora-21/lantw44-arm-linux-gnueabi-toolchain-fedora-21.repo && yum install -y arm-linux-gnueabi-gcc arm-linux-gnueabi-binutils arm-linux-gnueabi-glibc && yum clean all
+RUN yum install -y yum-utils && \
+  yum-config-manager --add-repo=https://copr.fedoraproject.org/coprs/lantw44/arm-linux-gnueabi-toolchain/repo/fedora-21/lantw44-arm-linux-gnueabi-toolchain-fedora-21.repo && \
+  yum install -y arm-linux-gnueabi-gcc arm-linux-gnueabi-binutils arm-linux-gnueabi-glibc && \
+  yum clean packages
 
 # Requisites for windows.
-RUN yum install -y mingw32-gcc.x86_64 && yum clean all
-
-# Boostrapping Go for different platforms.
-RUN cd $GOROOT/src && CGO_ENABLED=1 GOOS=linux GOARCH=amd64 ./make.bash --no-clean
-RUN cd $GOROOT/src && CGO_ENABLED=1 GOOS=linux GOARCH=386 ./make.bash --no-clean
-RUN cd $GOROOT/src && CXX_FOR_TARGET=arm-linux-gnueabi-g++ CC_FOR_TARGET=arm-linux-gnueabi-gcc CGO_ENABLED=1 GOOS=linux GOARCH=arm GOARM=7 ./make.bash --no-clean
-RUN cd $GOROOT/src && CXX_FOR_TARGET=i686-w64-mingw32-g++ CC_FOR_TARGET=i686-w64-mingw32-gcc CGO_ENABLED=1 GOOS=windows GOARCH=386 ./make.bash --no-clean
-
-RUN cd $GOROOT/src && GOARCH=386 ./make.bash --no-clean
+RUN yum install -y mingw32-gcc.x86_64 && yum clean packages
 
 # Requisites for building Lantern on Linux.
-RUN yum install -y gtk3-devel libappindicator-gtk3 libappindicator-gtk3-devel && yum clean all
-RUN yum install -y pango.i686 pango-devel.i686 gtk3-devel.i686 gdk-pixbuf2-devel.i686 cairo-gobject-devel.i686 atk-devel.i686 libappindicator-gtk3-devel.i686 libdbusmenu-devel.i686 dbus-devel.i686 pkgconfig.i686 && yum clean all
+RUN yum install -y gtk3-devel libappindicator-gtk3 libappindicator-gtk3-devel && yum clean packages
+RUN yum install -y pango.i686 pango-devel.i686 gtk3-devel.i686 gdk-pixbuf2-devel.i686 cairo-gobject-devel.i686 \
+  atk-devel.i686 libappindicator-gtk3-devel.i686 libdbusmenu-devel.i686 dbus-devel.i686 pkgconfig.i686 && \
+  yum clean packages
 
 # Requisites for packing Lantern for Debian.
 # The fpm packer. (https://rubygems.org/gems/fpm)
-RUN yum install -y ruby ruby-devel make && yum clean all
+RUN yum install -y ruby ruby-devel make && yum clean packages
 RUN gem install fpm
 
 # Requisites for packing Lantern for Windows.
-RUN yum install -y osslsigncode mingw32-nsis && yum clean all
+RUN yum install -y osslsigncode mingw32-nsis && yum clean packages
 
 # Required for compressing update files
-RUN yum install -y bzip2 && yum clean all
+RUN yum install -y bzip2 && yum clean packages
 
 # Requisites for genassets.
-RUN yum install -y nodejs npm && yum clean all
+RUN yum install -y nodejs npm && yum clean packages
 RUN npm install -g gulp
```

Comment:
@fffw Good idea, I'm gonna move things that do not change that often to the beginning of the file.